src/pages/Contact.jsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
-import Iframe from 'react-iframe';
 import { Typography, withStyles } from '@material-ui/core';
 import PropTypes from 'prop-types';
+import Container from '@material-ui/core/Container';
+import Box from '@material-ui/core/Box';
+import Paper from '@material-ui/core/Paper';
+import { grey } from '@material-ui/core/colors';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import Grid from '@material-ui/core/Grid';
+import LocationMap from '../components/LocationMap';
+import Form from '../components/Form';
+
+const githubLogo = require('../media/github.png');
```

Comment:
Use Material Design Logos instead of uploaded images. The logos are responsive. See line 9 in MemberViewCard.jsx. Also, they follow the material design - so, whenever, there is a change in the design - it shall remain homogeneous